src/main/scala/org/constellation/domain/redownload/RedownloadService.scala

```diff
@@ -45,6 +53,11 @@ class RedownloadService[F[_]](
       (updated, ())
     }
 
+  def persistStoredSnapshot(hash: String, bytes: Array[Byte]): F[Unit] = for {
+    _ <- snapshotStorage.writeSnapshot(hash, bytes).value.void
+    _ <- snapshotStorage.readSnapshot(hash).value.void
```

Comment:
If you're not using the snapshot, there is no point to read it from disk and deserialize